src/screens/ChannelListScreen/elements/components/AnonymousIcon.js

```diff
@@ -14,11 +15,12 @@ const AnonymousIcon = ({color, emojiCode, size, withBorder = false}) => {
       borderColor: COLORS.white,
       display: 'flex',
       justifyContent: 'center',
-      alignItems: 'center'
+      alignItems: 'center',
+      overflow: 'hidden'
     },
     anonPostNotificationEmoji: {
       display: 'flex',
-      fontSize: size * 0.75,
+      fontSize: dimen.normalizeDimen(size / 2),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `alignItems: 'center'` and `overflow: 'hidden'` properties are added to the style object. The fontSize calculation in `anonPostNotificationEmoji` has been changed from `size * 0.75` to `dimen.normalizeDimen(size / 2)`. Make sure that the `dimen.normalizeDimen` function exists and works as expected. Also, ensure that these changes do not affect the layout or appearance of the component adversely.

```diff
+      alignItems: 'center',
+      overflow: 'hidden'
...
-      fontSize: size * 0.75,
+      fontSize: dimen.normalizeDimen(size / 2),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->